tests/Oryx.BuildImage.Tests/Python/CondaTests.cs

```diff
@@ -0,0 +1,241 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Oryx.BuildScriptGenerator.Common;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.BuildImage.Tests
+{
+    public class CondaTests : PythonSampleAppsTestBase
+    {
+        public DockerVolume CreateCondaSampleAppVolume(string sampleAppName) =>
+            DockerVolume.CreateMirror(Path.Combine(_hostSamplesDir, "python", "conda-samples", sampleAppName));
+
+        public CondaTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void CanBuildAppWhichHasRequirementsTxtFile()
+        {
+            // Arrange
+            var appName = "requirements";
+            var volume = CreateCondaSampleAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var appOutputDir = "/tmp/app-output";
+            var script = new ShellScriptBuilder()
+                .AddBuildCommand($"{appDir} -o {appOutputDir}")
+                .AddDirectoryExistsCheck($"{appOutputDir}/venv")
+                // Following command makes sure that this package 'matplotlib' is present
+                .AddCommand("source /opt/conda/etc/profile.d/conda.sh")
+                .AddCommand($"conda activate {appOutputDir}/venv")
```

Comment:
Curious are these commands already included in the image or we required to run these commands every time we build a conda app

Reply:
These commands are already part of the generated script too. Here we added this to make sure that a particular package is indeed installed within the virtual environment.

Reply:
so the nightly build/tests and validations tests have successed so that means corresponding tests were already removed ...